testing/load-testing/README.md

```diff
@@ -0,0 +1,48 @@
+# K6
```

Comment:
What do you think about adding a section that explains how a load test should be carried out? Some simple steps like:

1. Identify critic parts of your application such as endpoints that require heavy database access or do some heavy processing.

2. Define the number of users to use during your tests. For example: 50, 100, 500, 1000, 2000, etc. The idea is to find the maximum number of concurrent users your app can handle at a specific time.

3. Summarize results. Give a number under which the application would have a good, regular and bad performace.

What do you think?